client/a8c-for-agencies/sections/overview/sidebar/growth-accelerator/cta.tsx

```diff
@@ -0,0 +1,54 @@
+import { Button } from '@wordpress/components';
+import { external, Icon } from '@wordpress/icons';
+import clsx from 'clsx';
+import { useTranslate } from 'i18n-calypso';
+import { useCallback } from 'react';
+import useFetchScheduleCallLink from 'calypso/a8c-for-agencies/data/agencies/use-fetch-schedule-call-link';
+import { useDispatch } from 'calypso/state';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+import { savePreference } from 'calypso/state/preferences/actions';
+import { GROWTH_ACCELERATOR_REQUESTED_PREFERENCE } from '.';
+
+type Props = {
+	className?: string;
+	onRequestSuccess?: () => void;
+};
+
+export default function OverviewSidebarGrowthAcceleratorCta( {
```

Comment:
We are refactoring this into a separate component, allowing us to reuse it in the Onboarding tour modal as well.